src/open_inwoner/openzaak/tests/test_case_detail_redirects.py

```diff
@@ -0,0 +1,74 @@
+from django.contrib.messages import get_messages
+from django.test import TestCase, override_settings
+from django.urls import reverse
+
+from open_inwoner.openzaak.models import ZGWApiGroupConfig
+from open_inwoner.openzaak.tests.factories import ZGWApiGroupConfigFactory
+
+
+@override_settings(
+    ROOT_URLCONF="open_inwoner.cms.tests.urls",
+)
+class LegacyCaseDetailUrlRedirectTest(TestCase):
+    def setUp(self):
+        self.url_patterns = [
+            "case_detail",
+        ]
+
+    def test_legacy_url_redirects_to_only_api_group_prefix(self):
+        first = ZGWApiGroupConfigFactory()
```

Comment:
```suggestion
        api_group = ZGWApiGroupConfigFactory()
```
Same below for this test. `first` is misleading as there is no `second`